thoth/storages/ceph_cache.py

```diff
@@ -36,3 +37,17 @@ def retrieve_document_record(self, document_id: str) -> dict:
     def store_document_record(self, document_id: str, document: dict) -> str:
         """Store the given document record in the cache."""
         self.ceph.store_document(document, document_id)
+
+    def retrieve_document_ttl(self, document_id: str) -> float:
+        """Retrieve the TTL for a cached document."""
+        # Verify if the document is present in the cache
+        self.retrieve_document_record(document_id)
+
+        # No timezone is indicated to calculate the present datetime on purpose to get the correct time delta
+        # regardless of the environment where the data is retrieved from (production or stage)
```

Comment:
```suggestion
        # Uses UTC time to be environment agnostic (no timezone)
```
I don't think we should point out specifics environments, thoth could be deployed on others.